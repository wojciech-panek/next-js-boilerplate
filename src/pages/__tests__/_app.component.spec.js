import React from 'react';
import { shallow } from 'enzyme';

import App from '../_app';


describe('App: Page', () => {
  const defaultProps = {
  };

  const component = (props) => (
    <App {...defaultProps} {...props} />
  );

  it('should render App root', () => {
    const wrapper = shallow(component());
    global.expect(wrapper).toMatchSnapshot();
  });
});
