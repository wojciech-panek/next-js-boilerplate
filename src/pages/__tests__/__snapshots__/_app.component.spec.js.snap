// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App: Page should render App root 1`] = `
<CustomApp
  store={
    Object {
      "dispatch": [Function],
      "execSagaTasks": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "runSaga": [Function],
      "saga": Object {
        "@@redux-saga/TASK": true,
        "cancel": [Function],
        "cont": undefined,
        "done": Promise {},
        "error": [Function],
        "id": 1,
        "isAborted": [Function],
        "isCancelled": [Function],
        "isRunning": [Function],
        "joiners": Array [],
        "name": "rootSaga",
        "result": [Function],
        "setContext": [Function],
      },
      "stopSaga": [Function],
      "subscribe": [Function],
      Symbol(Symbol.observable): [Function],
    }
  }
/>
`;
